bench/index.js

```diff
@@ -0,0 +1,148 @@
+const instances = require("./instances");
+const util = require("./util");
+const mini = require("minimist");
+const { adt } = require("@masaeedu/adt");
+const { Obj, Maybe, Either, Arr } = require("../src");
+const { Just, Nothing } = Maybe;
+const { Left, Right } = Either;
+
+const Command = adt({
+  RunAll: ["p"],
+  RunSuite: ["n", "s", "p"],
```

Comment:
Is this supposed to hold some specific types or just arbitrary things?

Reply:
Specific things; addressed in 6c5a0e4.